manifests/0000_26_cloud-controller-manager-operator_19_networkpolicy.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: ccmo-network-policy
+  namespace: openshift-cloud-controller-manager-operator
+spec:
+  podSelector: {}
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:
+    - from:
+      - podSelector:
+          matchLabels:
+            apiserver: 'true'
+            prometheus: k8s
+  egress:
+    - to:
+      - podSelector:
+          matchLabels:
+            apiserver: 'true'
+            prometheus: k8s
```

Comment:
Nit, all files should finish with a new line character (you can configure your IDE to do this).

If you don't it can lead to weird git diffs down the line when we modify the end of this file